cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+// Get environment variables
+const endPointUrl = Cypress.env('OSD_TEST_DOMAIN_ENDPOINT_URL');
+const invalidEndPointUrl = Cypress.env('OSD_INVALID_ENPOINT_URL');
+const username = Cypress.env('username');
+const password = Cypress.env('password');
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('Create datasources', () => {
+    beforeEach(() => {
+      // Visit OSD create page
+      miscUtils.visitPage(
+        'app/management/opensearch-dashboards/dataSources/create'
+      );
+    });
+
+    it('should successfully load the page', () => {
+      cy.contains(
+        'Create a new data source connection to help you retrieve data from an external OpenSearch compatible source.',
+        { timeout: 60000 }
+      );
+    });
+
+    it('should successfully show the experimental feature callout', () => {
+      cy.contains('Experimental Feature').should('exist');
+      cy.contains(
+        'The feature is experimental and should not be used in a production environment. Any index patterns, visualization, and observability panels will be impacted if the feature is deactivated. For more information see Data Source Documentation(opens in a new tab or window)To leave feedback, visit OpenSearch Forum'
+      ).should('exist');
+    });
+
+    describe('Datasource can be created successfully', () => {
```

Comment:
looks like the following 4 tests share same code, is it possible to make a data input/output array and run tests again each set of data using for loop?
Please check out [the Dynamic tests examples from Cypress](https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/fundamentals__dynamic-tests)